From f12e2ad41bb0c38f9dd6d90345c6f6ef20263359 Mon Sep 17 00:00:00 2001
From: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
Date: Mon, 25 Oct 2021 16:51:47 +0300
Subject: [PATCH 79/88] tty: rpmsg: Add pr_fmt() to prefix messages

Make all messages to be prefixed in a unified way.
Add pr_fmt() to achieve this.

Upstream-Status: Backport [88af70be4a5b936aa440bb9e22e6c57304067db6]
Signed-off-by: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
Link: https://lore.kernel.org/r/20211025135148.53944-4-andriy.shevchenko@linux.intel.com
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/tty/rpmsg_tty.c | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/drivers/tty/rpmsg_tty.c b/drivers/tty/rpmsg_tty.c
index 091862a057b37..277691040446d 100644
--- a/drivers/tty/rpmsg_tty.c
+++ b/drivers/tty/rpmsg_tty.c
@@ -10,6 +10,8 @@
  * The "rpmsg-tty" service is directly used for data exchange. No flow control is implemented yet.
  */
 
+#define pr_fmt(fmt)		KBUILD_MODNAME ": " fmt
+
 #include <linux/module.h>
 #include <linux/rpmsg.h>
 #include <linux/slab.h>
@@ -235,13 +237,13 @@ static int __init rpmsg_tty_init(void)
 
 	ret = tty_register_driver(rpmsg_tty_driver);
 	if (ret < 0) {
-		pr_err("Couldn't install rpmsg tty driver: %d\n", ret);
+		pr_err("Couldn't install driver: %d\n", ret);
 		goto error_put;
 	}
 
 	ret = register_rpmsg_driver(&rpmsg_tty_rpmsg_drv);
 	if (ret < 0) {
-		pr_err("Couldn't register rpmsg tty driver: %d\n", ret);
+		pr_err("Couldn't register driver: %d\n", ret);
 		goto error_unregister;
 	}
 
-- 
2.42.0

